deployment/scripts/run_dcl_node

```diff
@@ -0,0 +1,459 @@
+    dclcli config indent true
+    dclcli config trust-node false
+    if [[ -n "$_peer_host" ]]; then
+        dclcli config node tcp://"$_peer_host"
+    fi
+}
+
+
+function add_key {
+    set -eu
+    set -o pipefail
+
+    local _key_name="$1"
+    local _output="${2:-${_key_name}.dclkey.json}"
+
+    dclcli keys add "$_key_name" 2>&1 | tee "$_output"
+    echo -e "\tSaved key data in '$_output'"
+}
+
+
+function node_init {
+    set -eu
+    set -o pipefail
+
+    local _name="$1"
+    local _chain_id="$2"
+
+    rm -rf "$HOME/.dcld/config/gentx"
+    rm -f "$GENESIS_FILE"
+
+    dcld init "$_name" --chain-id "$_chain_id"
+}
+
+
+function build_genesis {
+    set -eu
+    set -o pipefail
+
+    local _key_name="$1"
+    local _acc_address="$2"
+    local _acc_pubkey="$3"
+    local _acc_roles="${4:-Trustee,NodeAdmin}"
+
+    dcld add-genesis-account --address="$_acc_address" --pubkey="$_acc_pubkey" --roles="$_acc_roles"
+    dcld gentx --from "$_key_name"
+    dcld collect-gentxs
+    dcld validate-genesis
+}
+
+function config_node {
+    set -eu
+    set -o pipefail
+
+    local _node_user="$1"
+    local _dcld_service_p="${2:-./dcld.service}"
+    local _peers="${3:-}"
+
+    local _debug="${DEBUG:-}"
+
+    sed -i -r 's~^laddr = "tcp://127.0.0.1:~laddr = "tcp://0.0.0.0:~' "$CONFIG_FILE"
+
+    # FIXME verify
+    if [[ -n "$_peers" ]]; then
+        sed -i -r "s~^persistent_peers = .*~persistent_peers = \"$_peers\"~" "$CONFIG_FILE"
+    fi
+
+    if [[ -z "$_debug" ]]; then
+        # config the service
+        # Ubuntu only
+        grep -q -i -e 'ubuntu' /etc/*-release && {
+            if [[ ! -f "$_dcld_service_p" ]]; then
+                echo "Error. '$_dcld_service_p' file not found"
+                exit 1
+            fi
+
+            # Open '26656' (p2p) and '26657' (RPC) ports. 
+            sudo ufw allow 26656/tcp
+            sudo ufw allow 26657/tcp
+            # set systemd dcld.service
+            sed -i -r "s~^User=ubuntu$~User=${NODE_USER}~" "$_dcld_service_p"
+            sudo cp -f "$_dcld_service_p" /etc/systemd/system/dcld.service
+        } || { echo "Warning: Not an Ubuntu system. Skipping systemd service configuration";}
+    fi
+}
+
+    
+function run_node {
+    set -eu
+    set -o pipefail
+
+    local _debug="${DEBUG:-}"
+
+    if [[ -z "$_debug" ]]; then
+        # Ubuntu only
+        grep -q -i -e 'ubuntu' /etc/*-release && {
+            # Enable the service
+            sudo systemctl enable dcld
+            # Start node
+            sudo systemctl start dcld
+        } || { echo "Warning: Not an Ubuntu system. Skipping service start..."; return 0; }
+
+        echo "Node has been stared as a service."
+
+        echo -e "\tUse 'systemctl status dcld' to get the node service status."
+        echo "Use 'journalctl -u dcld.service -f' to see node logs."
+    else
+        dcld start &
+        echo "Node has been stared in the backgroud."
+    fi
+}
+
+
+function verify_node {
```

Comment:
Are these functions used?

Reply:
no for now, there is a separate script that may help to do some checks but out of the current script logic